Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV GOOS=linux \
 
 # this value changes in ./bin/build
 ARG TAG="dev"
```

Comment:
where is `TAG` used? do we still need it?

Reply:
Its being used for tagging the container image. you are correct that we dont need it in the docker file.. deleted